pyproject.toml

```diff
@@ -51,34 +51,34 @@ djangorestframework-xml = "^2.0.0"
 drf-dynamic-fields = "*"
 feedparser = "^6.0.8"
 httplib2 = "^0.22.0"
-igraph = "^0.10.4"
-internetarchive = "^3.3.0"
+igraph = "^0.11.3"
+internetarchive = "^3.5.0"
 ipaddress = "^1.0.16"
 itypes = "^1.1.0"
 kombu = "^5.3.2"
 lxml = "==4.*"
 markdown2 = "==2.4.*"
-natsort = "^8.3.1"
+natsort = "^8.4.0"
 ndg-httpsclient = "^0.5.1"
-networkx = "^3.1"
+networkx = "^3.2.1"
 nose = "*"
 openapi-codec = "^1.3.1"
-pandas = "^1.5.0"
+pandas = "^2.1.3"
 pillow = "*"
 pycparser = "^2.21"
 pyopenssl = "*"
-pyparsing = "^3.1.0"
+pyparsing = "^3.1.1"
 python = ">=3.11, <3.12"
 python-dateutil = "^2.8.1"
-python-magic = "^0.4.21"
+python-magic = "^0.4.27"
 pytz = "*"
-pyyaml = "^6.0"
-redis = "^3.5.3"
+pyyaml = "^6.0.1"
+redis = "^5.0.1"
 requests = "^2.31.0"
-simplejson = "^3.18.3"
-stripe = "^5.2.0"
+simplejson = "^3.19.2"
+stripe = "^7.7.0"
```

Comment:
Changelog: https://github.com/stripe/stripe-python/blob/master/CHANGELOG.md

Reply:
Stripe has a bunch of breaking changes, and I think we're going to remove it soon. Can we remove it from the upgrade list?